### ISSUE TITLE
Confusing "Not found" log statement in swarm manager

### ISSUE THREAD

**Issue by ahmetalpbalkan** [docker/swarm#1077]
I'm using swarm:0.2.0 (although it reproes in 0.3.0 as well) and for some reason `swarm manage` does not report any healthy nodes...

join command:

    docker run -d swarm --debug join --addr=10.0.0.7:2375 token://1d7a047bf4787c7bdcf4ff7c065dae46

manage command:

    docker run -d -p 5555:2375 swarm --debug manage token://1d7a047bf4787c7bdcf4ff7c065dae46

In the logs of `swarm manage` container
```
time="2015-07-20T18:55:26Z" level=debug msg="Registering HTTP route" method=OPTIONS route=
time="2015-07-20T18:55:26Z" level=info msg="Listening for HTTP" addr=":2375" proto=tcp
time="2015-07-20T18:57:26Z" level=error msg="Not found"
```

the "Not found" line is sort of hard to trace. It should be more detailed to debug this

 When I try to create a container against the swarm manager, it says **there are no healthy nodes in the cluster** (`docker info` tells the same). It's probably because of this error?

The manager node and slave node are in separate VMs but are in the same Virtual Network (vlan). I verified that on the manager node, I can talk to the slave node using `docker -H tcp://10.0.0.7:2375 ps`.

Any ideas what that obscure `Not found` means? It's probably causing the node from being discovered.

**Comment by vieux** [comment]
@ahmetalpbalkan maybe it's because swarm runs in a container.

You can `docker -H tcp://10.0.0.7:2375 ps` from the manager machine, but can you 

```
docker run busybox nc 10.0.0.7 2375
GET /info HTTP/1.1
```

?

**Comment by ahmetalpbalkan** [comment]
@vieux yep looks like it's for some reason about outbound networking... Creating a new set of VMs solved the issue. :+1: 

Any ideas where the error might be coming through? It might be good to add some details or prefix it with the package name that returns it for further debugging perhaps?

**Comment by vieux** [comment]
the error is defined in dockerclient. We should update it so it's more explicit.

**Comment by ahmetalpbalkan** [comment]
hmm is it this: https://github.com/samalba/dockerclient/blob/917647f05ae67447538651120adcc14f79447108/dockerclient.go#L104-L106 if so it makes me wonder what was it that it actually responded fine but it was a 404. I just deleted the repro unfortunately.. :(

**Comment by abronan** [comment]
Will close this one, this is likely fixed by #1340 and swarm `0.2.0` is quite outdated now. @ahmetalpbalkan Let us know if you still hit this issue with `1.0.0` or feel free to open another one if necessary :)
